src/odemis/util/img.py

```diff
@@ -466,8 +465,28 @@ def rescale_hq(data, shape):
       information that is linked to the size (e.g, pixel size), it is also
       updated.
     """
```

Comment:
Need to update the shape description: needs to be the same size as the array.shape.
The "data" argument (and return) can be of any shape.